.github/workflows/generate.yml

```diff
@@ -0,0 +1,30 @@
+on:
```

Comment:
Can we name this file `wasm.yml`? I feel like `generate` is too generic 😅

Reply:
done